### PR TITLE
docs/usage.md: point to bridged networks in-tree

### PR DESCRIPTION
**Pull request by Mikaela**
Resolves: #1449

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ server administrator if the above defaults are not working for you. Server admin
 [config.sample.yaml](https://github.com/matrix-org/matrix-appservice-irc/blob/develop/config.sample.yaml) for
 instructions on how to change the templates for users and channels.
 
-The wiki contains a [list of public IRC networks](https://github.com/matrix-org/matrix-appservice-irc/wiki/Bridged-IRC-networks)
-including alias and user_id formats.
+Alias and user_id formats can be found from the [list of bridged networks](./bridged_networks.md)
```
Comment (Half-Shot) [inline docs/usage.md:35]: Nit, we prefer MXID

```suggestion
Alias and MXID formats can be found from the [list of bridged networks](./bridged_networks.md)
```

**Review by Half-Shot — APPROVED** [review]
Aha, thank you! Could you just do a sign off + newsfile and then we can merge this. https://github.com/matrix-org/matrix-appservice-irc/blob/develop/CONTRIBUTING.md#making-pull-requests

**Comment by Mikaela** [comment]
I did both and updated the word user_id to MXID, I had copied it from the old one.

**Comment by Half-Shot** [comment]
Looks great! thanks!

**Pull request merged**